client/signup/steps/site-or-domain/index.jsx

```diff
@@ -50,27 +54,49 @@ export default class SiteOrDomain extends Component {
 	}
 
 	getChoices() {
-		return [
+		const { translate } = this.props;
+
+		const choices = [
 			{
 				type: 'page',
-				label: 'New site',
+				label: translate( 'New site' ),
 				image: <NewSiteImage />,
-				description: 'Choose a theme, customize, and launch your site. Free domain included with all plans.'
-			},
+				description: translate( 'Choose a theme, customize, and launch your site. Free domain included with all plans.' )
+			}
+		];
+
+		if ( this.props.isLoggedIn ) {
+			choices.push(
+				{
+					type: 'existing-site',
+					label: translate( 'Existing WordPress.com site' ),
+					image: <ExistingSite />,
+					description: translate( 'Use with a site you already started. Free domain included with all plans.' )
+				}
+			);
+		}
+
+		choices.push(
 			{
 				type: 'domain',
-				label: 'Just buy a domain',
+				label: translate( 'Just buy a domain' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Buy Domain' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).